### DISCUSSION TITLE
Cannot import multiprocessing

### DISCUSSION THREAD

**Discussion by Moosems** [exaloop/codon#582] [General]
Installed with `/bin/bash -c "$(curl -fsSL https://exaloop.io/install.sh)"`, version is `0.17.0`, python version installed is `3.12` from python.org (not sure if that matters here) and the following reproduces the issue I'm seeing:
```bash
>>> echo "from multiprocessing import Process" >> issue.py; codon build issue.py
issue.py:2:6-21: error: no module named 'multiprocessing'
```

Any thoughts?

**Comment by avitkauskas** [comment]
`multiprocessing` module is not implemented natively in codon, so you should do:
`from python import multiprocessing` to use it.
Also, take a look at codon multiprocessing options here:
https://docs.exaloop.io/codon/advanced/parallel
